subscription/controls.py

```diff
@@ -371,12 +379,13 @@ def chargeForCyVerse(stripe_api_key, stripe_token, priceToCharge, stripeDescript
             except RuntimeError as error:
                 msg = "Your order has been processed, and your CyVerse account will be credited within 48 hours."
                 message['message'] = error
+                errMsg = message['message']
                 PaymentControl.sendCyVerseSyncFailedEmail(termLabel, username, purchaseDate, transactionId, error)
 
             PaymentControl.sendCyVerseEmail(msg, purchaseId, termLabel, partnerObj, emailAddress, firstname, lastname, priceToCharge, institute, transactionId, expirationDate, cardLast4, other)
         else:
-            #send email to tech team for an error in purchase (possible duplicate)
-            PaymentControl.sendCyVersePaymentErrorEmail(username, charge_amount, message)
+            #Covers the payment failure case
+            PaymentControl.sendCyVersePaymentErrorEmail(username, charge_amount, errMsg)
 
         if 'message' in message:
```

Comment:
Overall the update is good! The only update is here we want to log based on standard
if errMsg != '':